monitor/tests/mocks/mock_monasca.py

```diff
@@ -33,12 +36,13 @@ def __init__(self):
     Args:
         metrics(Objects): Object that must be send to monasca.
     """
+
     def send_metrics(self, metrics):
 
         try:
             self.metrics[metrics[0]['name']].append(metrics)
 
-        except Exception as ex:
```

Comment:
What is the main purpose of this modification? Can you please document this refactor in the description of this PR?